docs/spectral_cube.rst

```diff
@@ -2,79 +2,141 @@
+                                             5002.64862421] Angstrom>,
+                uncertainty=InverseVariance([[[3449.242 , 2389.292 , 2225.105 ],
+                      [4098.7485, 2965.88  , 2632.497 ],
+                      [3589.92  , 2902.7622, 2292.3823]],
+
+                     [[3563.3342, 2586.58  , 2416.039 ],
+                      [4090.8855, 3179.1702, 2851.823 ],
+                      [4158.919 , 3457.0115, 2841.1965]],
 
-We see that the example cube is small, with a shape of ``(7, 3, 4)``; two
-spatial axes ``RA`` and ``DEC``, and one spectral axis that's in velocity
-units.
+                     [[3684.6013, 3056.2   , 2880.6592],
+                      [3221.7888, 2801.3518, 2525.541 ],
+                      [3936.68  , 3461.534 , 3047.6135]]]))>
 
-Extracting the spectral data
-============================
 
-Now that we have a data cube, our mission is to collapse this cube along the
-two spatial axes to get the representative spectral data. We can then parse the
-result into a :class:`~specutils.Spectrum1D` object. ``SpectralCube`` can take
-and apply arbitrary functions over axes of the data, in this case we can apply
-a simple median function over the spatial axes::
+Spectral Bounding Region
+========================
 
-    >>> import numpy as np
-    >>> sc_spec = sc.apply_numpy_function(np.mean, axis=(1,2), projection=True)
-    >>> sc_spec.shape
-    (7,)
+The `~specutils.manipulation.extract_bounding_spectral_region` function can be used to
+extract the bounding region that encompases a set of disjoint `~specutils.spectra.SpectralRegion`
+instances, or a composite instance of `~specutils.spectra.SpectralRegion` that contains
+disjoint sub-regions.
 
-The ``projection`` keyword returns a lower dimesional object that retains the WCS
-information that we'll need to use when constructing the `~specutils.Spectrum1D` object.
+.. code-block:: python
 
-Creating a spectrum object
-==========================
+    >>> from specutils import SpectralRegion
+    >>> from specutils.manipulation import extract_bounding_spectral_region
+    >>> composite_region = SpectralRegion([(5000*u.AA, 5002*u.AA), (5006*u.AA, 5008.*u.AA)])
+    >>> sub_spectrum = extract_bounding_spectral_region(sc, composite_region)
+    >>> sub_spectrum.spectral_axis  #doctest:+SKIP
+    [5000.3453, 5001.4969, 5002.6486, 5003.8007, 5004.953, 5006.1055, 5007.2584]A˚
 
-With our data parsed and extracted, let's go ahead and create our
-:class:`~specutils.Spectrum1D` object::
 
-    >>> from specutils import Spectrum1D
-    >>> spec = Spectrum1D(flux=sc_spec.data[:] * sc_spec.unit, wcs=sc_spec.wcs)
+Moments
+=======
 
-Let's plot and see what this resulting spectrum looks like::
+The `~specutils.analysis.moment` function can be used to compute moments of any order
+along one of the cube's axis. By default, ``axis=2`` (or alternatively, ``axis=-1``)
+is used to compute moments along the spectral axis.
```

Comment:
```suggestion
along one of the cube's axes. By default, ``axis=-1``, which computes moments 
along the spectral axis (remember that the spectral axis is always last in a 
:class:`~specutils.Spectrum1D`).
```
I think we can drop the ``axis=2``, since it's actually `-1` in the code. I thought the reminder that the spectral axis is always last would be useful to include here, but if it seems extraneous feel free to remove it.